src/content/components/FlameChartViewport.js

```diff
@@ -1,9 +1,26 @@
-import React, { Component, PropTypes } from 'react';
+// @flow
+import React, { Component } from 'react';
 import FlameChartCanvas from './FlameChartCanvas';
+import type { Thread } from '../../common/types/profile';
+import type { Milliseconds, CssPixels, UnitIntervalOfProfileRange } from '../../common/types/units';
+import type { StackTimingByDepth } from '../stack-timing';
+
+type Props = {
+  connectedProps: {
```

Comment:
Ok, I took a look at it again this morning after a good night of sleep. This is because I want to pass in the props from the connected component to the FlameChartCanvas. I added the FlameChartViewport between the two components, and didn't want to enumerate a bunch of arbitrary values in the Viewport, because the Viewport doesn't care about that at all. Would there be a better way to approach this, or maybe a way to make this more clear?